[Export] Support aten.full.default and aten.full_like.default

sidt-meta opened the pull request:
Summary: Add operator tests for full & full_like operators

Test Plan:
Rerun kernel test using
```
buck2 run //glow/fba/tests:run_kernel mode/dev -- --kernel splat --config "input=1;dtype=fp32;fill_value=42.0"  -tl_time
```
{F1752274071}

Operator tests
```
buck2 run mode/{opt,inplace} //caffe2/torch/fb/test_library:afg_operator_test -- -k __full__
```
{F1752340913}

Differential Revision: D59593849

StellarrZ reviewed (approved):
LGTM! Please address the linter's suggestion

sidt-meta committed:
[Export] Support aten.full.default and aten.full_like.default (#130639)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/130639

Add operator tests for full & full_like operators

Test Plan:
Rerun kernel test using
```
buck2 run //glow/fba/tests:run_kernel mode/dev -- --kernel splat --config "input=1;dtype=fp32;fill_value=42.0"  -tl_time
```
{F1752274071}

Operator tests
```
buck2 run mode/{opt,inplace} //caffe2/torch/fb/test_library:afg_operator_test -- -k __full__
```
{F1752340913}

Reviewed By: StellarrZ

Differential Revision: D59593849
